LuckyPills/EventHandlers.cs

```diff
@@ -130,8 +99,8 @@ private IEnumerator<float> RunPillCoroutine(UsingMedicalItemEventArgs ev)
         {
           var cachedMutatorRole = ev.Player.Role;
           ev.Player.DropItems();
-          ev.Player.SetRole(RoleType.Scp0492, true);
-          Timing.CallDelayed(num, () => ev.Player.SetRole(cachedMutatorRole, true));
+          ev.Player.SetRole(RoleType.Scp0492);
```

Comment:
It was lite for a reason. The zombies will spawn in a black void.

Reply:
they do i tested it